recipe/meta.yaml

```diff
@@ -24,7 +24,8 @@ source:
 
 build:
   number: 1  # [win]
-  number: 3  # [unix]
+  number: 3  # [linux]
+  number: 4  # [osx]
```

Comment:
@stuarteberg I can merge this but we need a volunteer to build the package in a Mac so we can upload it.